packages/ui/src/views/connectors/connectors-list/connectors-list.tsx

```diff
@@ -12,7 +12,7 @@ import {
   Text,
   Tooltip
 } from '@/components'
-import { timeAgo } from '@utils/utils'
+import { timeAgo } from '@/utils'
```

Comment:
none of these import changes are required in this PR. lets remove it.

Reply:
Explained in [this thread](https://pixelpointworkspace.slack.com/archives/C084FDWFA21/p1746785725188729?thread_ts=1746699952.109979&cid=C084FDWFA21)